libpromises/evalfunction.c

```diff
@@ -1563,8 +1563,45 @@ static FnCallResult FnCallVariablesMatching(EvalContext *ctx, ARG_UNUSED const P
 }
 
 /*********************************************************************/
+static Bundle *GetBundleFromPolicy(const Policy *policy, const char *namespace, const char *bundlename)
+{
+    const size_t bundles = SeqLength(policy->bundles);
 
-static FnCallResult FnCallGetMetaTags(EvalContext *ctx, ARG_UNUSED const Policy *policy, const FnCall *fp, const Rlist *finalargs)
+    for (size_t i = 0; i < bundles; i++)
+    {
+        Bundle *bp = SeqAt(policy->bundles, i);
+        if ((strcmp(bp->name, bundlename) || strcmp(bp->ns, namespace)) == 0)
+        {
+            return bp;
+        }
+    }
+    return NULL;
+}
+
+static Promise *GetPromiseFromBundle(const Bundle *bundle, const char *promise_type, const char *promiser)
+{
+    const size_t sections_length = SeqLength(bundle->sections);
+    for (size_t i = 0; i < sections_length; i++)
+    {
+        BundleSection *bsection = SeqAt(bundle->sections, i);
+        if (strcmp(bsection->promise_type, promise_type) == 0 )
+        {
+            const size_t promises_length = SeqLength(bsection->promises);
```

Comment:
Same here; `num_promises`

Reply:
This one is not the same as the others, this one says `_length`, that's nice and specific, arguably better than `num`.